<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(0, 92, 137);
	color: white
}
</string>
  </property>
  <widget class="QLineEdit" name="toLineEdit">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fromLineEdit">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateReportButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 16pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>تولید گزارش</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalIncomeLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalExpenseLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="netBalanceLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>790</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 11pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="categoryTable">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>441</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}

QHeaderView::section {
    background-color: #eeeeee;
    color: #333333;
    font-weight: bold;
    font-size: 13px;
    padding: 5px;
    border: 1px solid #cccccc;
}

</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="accountTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}

QHeaderView::section {
    background-color: #eeeeee;
    color: #333333;
    font-weight: bold;
    font-size: 13px;
    padding: 5px;
    border: 1px solid #cccccc;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>  تاریخ:        از</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>تا</string>
   </property>
  </widget>
  <widget class="QLabel" name="netBalanceLabel_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>مانده‌ خالص:</string>
   </property>
  </widget>
  <widget class="QLabel" name="netBalanceLabel_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>مجموع درآمد:</string>
   </property>
  </widget>
  <widget class="QLabel" name="netBalanceLabel_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>مجموع هزینه:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>گزارش درآمد/هزینه</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>گزارش مانده حساب ها</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>500</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>سهم دسته‌های هزینه</string>
   </property>
  </widget>
  <widget class="QWidget" name="incomeChartContainer" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>سهم دسته‌های درآمد</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportToExcelButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 16pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>خروجی اکسل</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 16pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>سال:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEdit">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="generateYearlyReportButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 16pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>تولید گزارش سالیانه</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportToExcelYearlyButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 16pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>خروجی اکسل سالیانه</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="categorycostTable">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>230</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}

QHeaderView::section {
    background-color: #eeeeee;
    color: #333333;
    font-weight: bold;
    font-size: 13px;
    padding: 5px;
    border: 1px solid #cccccc;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>گزارش دسته های هزینه</string>
   </property>
  </widget>
  <widget class="QWidget" name="TrendChartContainer" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="expenseChartContainer" native="true">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>560</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>روند مالی ماهانه</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="categoryincomeTable">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>230</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    color: #222222;
    gridline-color: #cccccc;
    font-size: 14px;
    font-family: Tahoma;
    selection-background-color: #d0eaff;
    selection-color: #000000;
    border: 1px solid #cccccc;
}

QHeaderView::section {
    background-color: #eeeeee;
    color: #333333;
    font-weight: bold;
    font-size: 13px;
    padding: 5px;
    border: 1px solid #cccccc;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: white;
    font: 20pt &quot;.AppleSystemUIFont&quot;;
}

</string>
   </property>
   <property name="text">
    <string>گزارش دسته های درآمد</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
